cli/src/cli.rs

```diff
@@ -2253,6 +2266,13 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                         .hidden(true) // Don't document this argument to discourage its use
                         .help("Use the deprecated BPF loader")
                 )
+                .arg(
+                    Arg::with_name("force_use_system_account")
+                        .long("force-use-system-account")
+                        .takes_value(false)
+                        .hidden(true) // Don't document this argument to discourage its use
+                        .help("Use the designated program id, even if it already holds a balance of SOL")
```

Comment:
I'm ambivalent about whether this should be hidden or not. Wdyt?

Reply:
Hidden seems right to me.

The _system_ in the name on the other hand confuses me a bit.  Maybe `--force-use-program-address` ?

Reply:
`--force-use-program-address` is certainly better. I was trying to indicate that the flag will only force by the one failure/error, but maybe that's an over-complication.

Reply:
IMO let's unhide it.   

`--force-use-program-address` gets 👎 from me due to `Pubkey::create_program_address()` existing.
`--allow-excessive-deploy-account-balance`?